### DISCUSSION TITLE
Use `squash and merge` for merging PR

### DISCUSSION THREAD

**Discussion by MicBun** [trufnetwork/node#97] [General]
Recently we merged commits with `Create a merge commit` This will make our commit history messy if there are a pull request with multiple commits. I suggest we use `squash and merge` to keep things neat. Another option here is to disable the merge commit and simply use squash instead to prevent accidental merge with multiple small commits. 

![image](https://github.com/truflation/tsn-db/assets/48527109/8ffdf814-0de5-4504-8daa-c4bb57b8da92)
![image](https://github.com/truflation/tsn-db/assets/48527109/ebd35882-8d03-4552-ad8a-16e708e7dd42)
![image](https://github.com/truflation/tsn-db/assets/48527109/76966a91-4161-49f6-8b51-b3454ea662e4)

**Comment by zolotokrylin** [comment]
Good point. I have changed the setting. Now only the Squash option should be available. Please check when there's a chance. Thanks
